src/pyggdrasil/tree_inference/_simulate.py

```diff
@@ -672,19 +670,13 @@ def gen_sim_data(
 
     # Save the data to a JSON file
     # Create a dictionary to hold matrices
-    if noisy_mutation_mat is not None:
-        data = {
-            "adjacency_matrix": tree_adj_mat.tolist(),
-            "perfect_mutation_mat": perfect_mutation_mat.tolist(),
-            "noisy_mutation_mat": noisy_mutation_mat.tolist(),
-            "root": root_serialized,
-        }
-    else:
-        data = {
-            "adjacency_matrix": tree_adj_mat.tolist(),
-            "perfect_mutation_mat": perfect_mutation_mat.tolist(),
-            "root": root_serialized,
-        }
+
+    data = {
+        "adjacency_matrix": tree_adj_mat.tolist(),
+        "perfect_mutation_mat": perfect_mutation_mat.tolist(),
+        "noisy_mutation_mat": noisy_mutation_mat.tolist(),
+        "root": root_serialized,
+    }
```

Comment:
Made the output of `cell_simulation.py` static. - Allways output the same dictionary with the `noisy_mutation_mat' even if noise is none. - Makes this a lot easier to handle.

Reply:
This is an excellent idea!